keyboards/projectcain/vault35/keymaps/default/config.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2021 projectcain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define ENCODERS_PAD_A { D6 }
+#define ENCODERS_PAD_B { D7 }

```

Comment:
Is there only one possible location for the encoder on this board? And where are the possible locations?

Reply:
Correct - one possible location for the 35. It is in the top right directly on top of a switch footprint so they're interchangeable